From a9927d6218044e586a7dff79c1fc081b0033338e Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Mon, 20 Jan 2025 14:04:38 +0100
Subject: [PATCH 1472/1695] scp: remove rejected hunk file

remove unnecessary file :
./core/lib/scmi-server/SCP-firmware/module/CMakeLists.txt.rej

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 8ae276db57bdd2934ec084b780eab647738ac1e7 scp: remove rejected hunk file]
Change-Id: Ic7cf7182479d0348ff760325050289d2a6232636
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/429981
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 .../SCP-firmware/module/CMakeLists.txt.rej       | 16 ----------------
 1 file changed, 16 deletions(-)
 delete mode 100644 core/lib/scmi-server/SCP-firmware/module/CMakeLists.txt.rej

diff --git a/core/lib/scmi-server/SCP-firmware/module/CMakeLists.txt.rej b/core/lib/scmi-server/SCP-firmware/module/CMakeLists.txt.rej
deleted file mode 100644
index 2033d30cd..000000000
--- a/core/lib/scmi-server/SCP-firmware/module/CMakeLists.txt.rej
+++ /dev/null
@@ -1,16 +0,0 @@
-diff a/core/lib/scmi-server/SCP-firmware/module/CMakeLists.txt b/core/lib/scmi-server/SCP-firmware/module/CMakeLists.txt	(rejected hunks)
-@@ -44,12 +44,14 @@ list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/mock_voltage_domain")
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/mpmm")
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/msg_smt")
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/msys_rom")
-+if(SCP_OPTEE_DIR)
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/optee/clock")
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/optee/console")
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/optee/mbx")
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/optee/reset")
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/optee/smt")
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/optee/voltd_regulator")
-+endif(SCP_OPTEE_DIR)
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/pcid")
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/pik_clock")
- list(APPEND SCP_MODULE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/pl011")
-- 
2.47.2

